spec.bs

```diff
@@ -1563,3 +1576,26 @@ specification adds another value for fenced frames:
 
 : "`fencedframe`"
 :: This [=navigable=] is displaying a <{fencedframe}>'s content
+
+<h3 id=local-network-access-monkeypatch>Local Network Access</h3>
+
+Local network access is a security feature that allows you to control which apps on your device can
+access your local network. Regular iframes inherit their policy container from the navigation
+initiator, while fenced frames explicitly do not. This results in a policy container being created
```

Comment:
> Regular iframes inherit their policy container from the navigation initiator

Is this true? When they are _created_ I _think_ that's the case, but otherwise their policy container is informed from the [response creation algorithm](https://html.spec.whatwg.org/C#creating-a-policy-container-from-a-fetch-response), right?

Reply:
Correct. This is just talking about the creation case. Updated the comment to explicitly specify that.